Source/JavaScriptCore/runtime/StructureCache.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2013-2019 Apple Inc. All rights reserved.
+ * Copyright (C) 2013-2023 Apple Inc. All rights reserved.
```

Comment:
Looks like there's no other change in this file.  So, this is not needed.

Reply:
Fixed.